Example/Firebase.xcodeproj/project.pbxproj

```diff
@@ -4303,13 +4304,14 @@
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				51559F8A2238B8DB00CFC32C /* FIRMessagingExtensionHelperTest.m in Sources */,
+				511DD2962225C8C40094D78D /* FIRMessagingClientTest.m in Sources */,
```

Comment:
This file is not in any of the test targets.

Reply:
Think I just added that to the project setting, still not working for you if you pull the latest?

Reply:
I believe you added it back to the project, but not to the Test build targets:

<img width="1429" alt="Screen Shot 2019-03-12 at 9 59 12 PM" src="https://user-images.githubusercontent.com/73870/54254756-3efce600-4512-11e9-9666-a89d1cb95d20.png">

Reply:
yep I missed that one file in the build target